packages/statusbar-extension/schema/plugin.json

```
@@ -30,6 +30,12 @@
       "title": "Status Bar Visibility",
       "description": "Whether to show status bar or not",
       "default": true
+    },
+    "startMode": {
+      "type": "string",
```

Let's use an enumeration.

```suggestion
      "enum": ["", "single", "multiple"],
```